modules/virt-assigning-pci-device-virtual-machine.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-assigning-pci-device-virtual-machine_{context}"]
+= Assigning a PCI device to a virtual machine
+
+When a PCI device is available in a cluster, you can assign it to a virtual machine and enable PCI passthrough.
+
+.Procedure
+. Assign the PCI device to a virtual machine as a host device.
++
+.Example
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+spec:
+  domain:
+    devices:
+      hostDevices:
+      - deviceName: nvidia.com/TU104GL_Tesla_T4 <1>
+        name: hostdevices1
+----
+<1> The name of the PCI device that is permitted on the cluster as a host device. The virtual machine can access this host device.
+
+.Verification
+* Use the following command to verify that the host device is available from the virtual machine.
++
+[source,terminal]
+$ lspci -nnk | grep NVIDIA
+
+.Example
+[source,output]
+$ 02:01.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 32GB] [10de:1eb8] (rev a1)
```

Comment:
This should be a code block (`----` on the top and bottom)

Reply:
Added code block.